R/data.table.R

```diff
@@ -213,8 +203,12 @@ replace_dot_alias = function(e) {
   av = NULL
   jsub = NULL
   if (!missing(j)) {
+    colselect = getOption("datatable.colselect", FALSE)
```

Comment:
How long would this be an option?

Reply:
I will write more on that when PR will be more mature. It has to be an option unfortunately, there are so many corner cases than it is very unlikely to address them maintaining backward compatibility.